examples/python/live-recognition/detect_barcodes.py

```diff
@@ -56,59 +60,68 @@ def recognize_live(recognizer:scanbotsdk.BarcodeRecognizer, device:str, use_disp
     if not cap.isOpened():
         print("Cannot open camera")
         exit(1)
-    
-    if use_display:
-        cv.namedWindow("BarcodeRecognizer", cv.WINDOW_AUTOSIZE)
-
-    while(True):
-        _, image = cap.read()
-        if image is None:
-            print("Failed to capture image")
-            break
-
-        barcodes = recognizer.recognize(image)
-        parse_barcodes(barcodes, image)
 
+    try:
         if use_display:
-            width = image.shape[1] * DISPLAY_HEIGHT / image.shape[0]
-            image = cv.resize(image, (int(width), DISPLAY_HEIGHT))
-            cv.imshow("BarcodeRecognizer", image)
-            cv.waitKey(1)
-
-            if cv.getWindowProperty("BarcodeRecognizer", cv.WND_PROP_AUTOSIZE) == -1:
+            cv.namedWindow("BarcodeRecognizer", cv.WINDOW_AUTOSIZE)
+        image = np.zeros((DEVICE_CAPTURE_HEIGHT, DEVICE_CAPTURE_WIDTH, 3), dtype=np.uint8)
+        while True:
+            # Note that the image is updated in-place
+            ret, _ = cap.read(image)
+            if not ret:
+                print("Failed to capture image")
                 break
-                    
-    cap.release()
+
+            # Although it is not required to use the image ref as a context manager, it is recommended to do so when
+            # creating from live source to ensure that the image_ref is timely released.
+            #
+            # !!! IMPORTANT !!! It is critically important to specify the live_source=True option when creating the
+            # image ref to run the scanner in live mode. As an alternative it is possible to specify
+            # processing_mode=ProcessingMode.LIVE in the scanner configuration.
```

Comment:
This is a bit over-dramatic. How about the following more neutral and more informative rewording:

> Setting `live_source=True` lets the scanner know that we're running in live mode. In this mode we maintain the highest FPS because we spread the work of scanning barcodes across multiple frames. If you set `live_source=False`, the scanner will run in single-shot mode which is much slower, but has a much higher probability of finding all barcodes in the input image. As an alternative to setting `live_source`, you can explicitly set the scanner mode, by creating it with e.g. `processing_mode=ProcessingMode.LIVE` for live mode.